model/metrics/vcs.yaml

```diff
@@ -0,0 +1,150 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: gauge
+    unit: "{change}"
+    stability: experimental
+    #extends: metric_attributes.vcs
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_open
+    type: metric
+    metric_name: vcs.repository.change.time_open
+    brief: 'The amount of time a change (pull request) has been open'
```

Comment:
is it cumulative? E.g. if I open and then close and then reopen, what would it count??

Reply:
That's an interesting question because it shows the gap between interval based scraping and event counting. These metrics originate from a scraping, that has to run through all the queries necessary to populate these metrics. As such, it's not looking back at history but solely looking at state, point in time. It depends on if the API in this case changes the opened date to be the re-opened date.

Reply:
Ideally it would be cumulative. It's the time from now to creation time - any time spent closed.
However this would likely be difficult to implement.
I would be fine with leaving it unspecified for now and create a follow up task.